### PR TITLE
Updateconfig plugin updates knative prow configs from oss-test-infra prow

### PR DESCRIPTION
**Pull request by chaodaiG**
This is step 4 as described in https://github.com/GoogleCloudPlatform/oss-test-infra/pull/407

Shouldn't merge until steps 1-3 are done
/hold

### CODE REVIEW

File: prow/oss/plugins.yaml [hunk 1]
```diff
@@ -20,6 +20,16 @@ config_updater:
       name: plugins
     prow/prowjobs/**/*.yaml:
       name: job-config
+    prow/knaitve/config.yaml: # For knative prow
+      name: config
+      clusters: 
+        knative-prow-trusted:
```
Comment (fejta) [inline prow/oss/plugins.yaml:26]: Where do these cluster keys come from?

Comment (chaodaiG) [reply prow/oss/plugins.yaml:26]: It'll be from `kubeconfig` secret, it'll need to be configured by oss-test-infra prow admin

Comment (fejta) [reply prow/oss/plugins.yaml:26]: The kubeconfig for scheduling jobs? This isn't scheduling jobs though.

Comment (chaodaiG) [reply prow/oss/plugins.yaml:26]: Yeah it's not for scheduling jobs, the `kubeconfig` is mentioned as documented in updateconfig plugin:
```
# specify the clusters and namespaces that the configmap targets
      # which requires that either --build-cluster or --kubeconfig arg is enabled for Hook
      # https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#run-test-pods-in-different-clusters
      # if not set or empty, it uses the cluster where prow components are running
      # and the specified namespace(s)
      # if defined, the above namespace will be ignored
      clusters: 
        others:
        - namespace1
```

My understanding is it reuses the same kubeconfig for hook
https://github.com/kubernetes/test-infra/tree/master/prow/plugins/updateconfig

File: prow/oss/plugins.yaml [hunk 2]
```diff
@@ -20,6 +20,16 @@ config_updater:
       name: plugins
     prow/prowjobs/**/*.yaml:
       name: job-config
+    prow/knaitve/config.yaml: # For knative prow
```
Comment (cjwagner) [inline prow/oss/plugins.yaml:23]: typo

Comment (chaodaiG) [reply prow/oss/plugins.yaml]: Fixed, thanks

**Comment by chaodaiG** [comment]
Everything is ready for test
/unhold

I'll monitor the situation

**Pull request merged**